### PR TITLE
Add Qt

### PR DESCRIPTION
**Pull request by JesseTG**
**https://github.com/JesseTG/awesome-qt**

**Qt is a popular framework for desktop (and recently mobile) application development.  Some well-known software uses it, too.  This lists includes libraries, tools, blogs, and books that are useful for beginners and experts alike.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title. For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.
- The list I added
	- **has been around for at least 30 days,**
	- is the result of hard work and is the best I could possibly produce,
	- is a non-generated Markdown file in a GitHub repo,
	- is not a duplicate,
	- includes a succinct description of the project/theme at the top ([example](https://github.com/willempienaar/awesome-quantified-self)),
	- includes a project logo/illustration (whenever possible) at the top right of the readme ([example](https://github.com/sindresorhus/awesome-electron)). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),
	- only has awesome items **with descriptions** *(Awesome lists are curations of the best, not of everything)*,
	- contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- has a Table of Contents section **named `Contents`** as the first section,
	- has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)* *(If you use a license badge, it should be SVG, not PNG)*,
	- has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- doesn't include a Travis badge,
	- has been checked for inconsistent formatting, spelling, and grammar:
		- Each link description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent naming. For example, `Node.js`, not `NodeJS` or `node.js`.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Nice work. Looks pretty good :)

---

> The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.

Not followed.

> includes a succinct description of the project/theme at the top (example),

Not followed.

> includes a project logo/illustration (whenever possible) at the top right of the readme (example). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),

Not followed.

---

Would be nice if the channel names in `and include #qt, #qt-quick, #qt-creator, and #qt-chat.` linked to the the channel on the Freenode webchat. For example: https://webchat.freenode.net/?channels=qt

**Comment by JesseTG** [comment]
> The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.
How's the one I just added?

> includes a succinct description of the project/theme at the top (example),
Is there a particular example you have in mind?

> includes a project logo/illustration (whenever possible) at the top right of the readme (example). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),
You mean the Qt logo, or a logo that *I* will make?

File: readme.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Fuse](https://github.com/vinkla/awesome-fuse)
 - [Heroku](https://github.com/ianstormtaylor/awesome-heroku)
 - [Raspberry Pi](https://github.com/thibmaek/awesome-raspberry-pi) - Credit card-sized computer aimed at teaching kids programming, but capable of a lot more.
-
+- [Qt](https://github.com/JesseTG/awesome-qt) - A powerful cross-platform GUI-centric application framework.
```
Comment (sindresorhus) [inline readme.md]: Don't remove the newline.

Could be more succinct:

> Cross-platform GUI app framework.

**Comment by sindresorhus** [comment]
> Is there a particular example you have in mind?

There's an example linked in the guidelines.

**Comment by JesseTG** [comment]
OK, how about now?

**Comment by sindresorhus** [comment]
> includes a project logo/illustration (whenever possible) at the top right of the readme (example). (The logo should link to the project website or any relevant website.)

Please read all the text.

**Comment by JesseTG** [comment]
Okay, revised the logo a bit.

**Comment by sindresorhus** [comment]
Looks good now :)

**Comment by JesseTG** [comment]
Thank you very much!

**Pull request merged**